web-ui/src/pages/BirthdayAnniversaryReportPage.jsx

```diff
@@ -47,6 +47,20 @@ const BirthdayAnniversaryReportPage = () => {
     setAnniversary(!anniversary);
   };
 
+  const sortBirthdays = (birthdayData) => {
+    return birthdayData.sort(
+      (a, b) =>
+        Number(
+          b.birthDay.substring(b.birthDay.indexOf("/"), b.birthDay.length)
+        ) -
+        Number(a.birthDay.substring(a.birthDay.indexOf("/"), a.birthDay.length))
+    );
+  };
```

Comment:
I don't think this will work if multiple months are selected, will it?

Reply:
I fixed this as it would cause issues sorting dates. I'm pushing the fix now.